nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -659,6 +673,16 @@ protected static void assertActorIs(String expectedActor, JsonNode taskNode) {
         assertEquals(expectedActor, actors.get(0));
     }
 
+    protected static void assertThatContainsActors(List<String> expectedActors, JsonNode actorsNode) {
+        Iterator<JsonNode> actorNode = actorsNode.elements();
+        List<String> actors = new ArrayList<>();
+        while (actorNode.hasNext()) {
+            actors.add(actorNode.next().get("id").textValue());
+        }
+        assertEquals(expectedActors.size(), actors.size());
+        assertThat(actors.containsAll(expectedActors), is(true));
```

Comment:
Why not simply `assertTrue(actors.containsAll(expectedActors));` and get rid of
```
import static org.hamcrest.Matchers.is;
import static org.junit.Assert.assertThat;
```
?

Reply:
It is only about some feelings related to assertTrue.
But yes, in this specific case, it would not be that bad!

Thanks!